File: test/fixtures/AssemblyInfo-modified.js
import System.Reflection;
import System.Runtime.CompilerServices;
import System.Runtime.InteropServices;

[assembly: AssemblyTitleAttribute("ModifiedTitle")]
[ assembly: AssemblyDescriptionAttribute('escape\'this"')]
[assembly:AssemblyConfigurationAttribute("")]
  [assembly: AssemblyCompanyAttribute("ModifiedCompany")]
[assembly: AssemblyProductAttribute("TestProduct" )]
[assembly: AssemblyCopyrightAttribute("Copyright © TestCompany 2017") ]
[assembly: AssemblyTrademarkAttribute("ModifiedTrademark")]
[
    assembly: AssemblyCultureAttribute(""
)]

// Boolean
[assembly: ComVisibleAttribute(false)]

// Attributes in comments are igored
// [assembly: AssemblyVersion("1.0.*")]
[assembly: AssemblyVersionAttribute("1.0.0")]

// In fact any other type of statement is ignored
whatever
  [assembly: InternalsVisibleToAttribute("FooBar")]
foobar
[assembly: NewAttrAAttribute(true)]
[assembly: NewAttrBAttribute("beep")]

File: test/fixtures/AssemblyInfo-modified.cs
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Runtime.InteropServices;

[assembly: AssemblyTitle("ModifiedTitle")]
[ assembly: AssemblyDescription('escape\'this"')]
[assembly:AssemblyConfiguration("")]
  [assembly: AssemblyCompany("ModifiedCompany")]
[assembly: AssemblyProduct("TestProduct" )]
[assembly: AssemblyCopyright("Copyright © TestCompany 2017") ]
[assembly: AssemblyTrademark("ModifiedTrademark")]
[
    assembly: AssemblyCulture(""
)]
[assembly: SuppressIldasm]

// Boolean
[assembly: ComVisible(false)]

// Attributes in comments are igored
// [assembly: AssemblyVersion("1.0.*")]
[assembly: AssemblyVersion("1.0.0")]

// In fact any other type of statement is ignored
#if DEBUG
  [assembly: InternalsVisibleTo("FooBar")]
#endif
[assembly: NewAttrA(true)]
[assembly: NewAttrB("beep")]
